src/math/p5.Vector.js

```diff
@@ -3901,6 +3901,22 @@ p5.Vector = class {
  * @method clampToZero
  * @return {p5.Vector} with components very close to zero replaced with zero.
  * @chainable
+ * @example
+ * <div">
```

Comment:
do we want div> instead of div"> .... in the second one we have extra quotation marks